test/scripts/stage.inc

```diff
@@ -0,0 +1,44 @@
+#
+# Include for common parts for ephemeral environment shared between all the scripts
+#
+# NOTE: Be aware that curl.sh wrapper set options based in the environment
+#       variables that has value when it is invoked, and set an environment
+#       variable could change the behave on how the request is formed.
+#
+# See: ./scripts/curl.sh
+#
+source "$(dirname "${BASH_SOURCE[0]}")/common.inc"
+
+NAMESPACE="idmsvc-stage"
+export NAMESPACE
+
+if [ "${https_proxy}" == "" ] && [ "${http_proxy}" == "" ]; then
+    error "https_proxy and http_proxy are empty: check your proxy configuration"
```

Comment:
Using a proxy might not be always needed - it depends on where the script is run. E.g. it is needed when using vpn.

Could the script advise what to set when it will fail because of missing proxy info?